cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_hgvs.py

```diff
@@ -693,8 +693,30 @@ def get_updated_hgvs_transcripts(annovar_transcripts: list, pysam_file):
                              merged_transcripts,
                              best_transcripts))
 
-    return best_transcripts
+    
+    # HGVS convention is to place predicted consequences in parenthesis but our users don't care for the parenthesis so they are removed. 
+    return _strip_pdot_parenthesis(best_transcripts)
```

Comment:
The question that needs to be answered before merging this pull request is - is this the best place to remove the parenthesis from the p-dots?  
* Here, I am calling  `` _strip_pdot_parenthesis(list)`` on the list of all variant-transcripts, which costs O(n). 
    * I did it this way because I originally planned on having a command line parameter that let the caller indicate whether or not to strip the parenthesis.  
* This could be done somewhere else at the same time the pdot is being set.  It's a little trickier than that because pDot gets set in multiple places, by annovar and hgvs.  
    *